### ISSUE TITLE
After game ending (bungee) the server is stopped -> config for /restart instead

### ISSUE THREAD

**Issue by hyperverse** [Yannici/bedwars-reloaded#174]
Hello,
I have a suggestion: Using bungeecord and spigot for the map, it would be nice to allow for a spigot /restart command after game ending. 
I am using mark2 on linux and with the restart-script from spigot this would allow an easy server restart after each game.
Could you make it an option in the config.yml to use spigots /restart instead of /stop?
And... thanks for this great plugin!

**Comment by Yannici** [comment]
I'll check if Spigot is used, then "/restart" will be executed. Otherwise it will stop the server.

**Comment by KillaSeven** [comment]
I think this, in combination with my reported bug, is a great help for spigot + bungeecord users.

**Comment by Yannici** [comment]
I'm working on it ;)

**Comment by Yannici** [comment]
Added in https://github.com/Yannici/bedwars-reloaded/commit/22cc22c4b3bc1de3a6390015b1825948f416bf5c

**Comment by hyperverse** [comment]
Hi Yannici,
thanks for implementing this so quickly.
I tried out the 1.2 jar and there is an issue. It think the plugin is sending "/restart" to the console instead of "restart". 
-> when I do a spigot alias "/restart" to "restart", then it works.

Another issue I recognized in 1.2 is that the en locale has missing entries compared to de so at game end in the display it gives "locale not found" instead of the winning message.
